qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/diagnostics/QuteErrorCode.java

```diff
@@ -66,6 +66,8 @@ public enum QuteErrorCode implements IQuteErrorCode {
 	UnexpectedMemberTypeInCaseSection("Unexpected type `{0}` in `{1}`. Expected `{2}`."),
 	UnexpectedValueInCaseSection("Unexpected value `{0}` in `{1}`. Expected value of type `{2}`."),
 	InvalidParentInCaseSection("`{0}` section must be hosted in a #switch or #when section."),
+	UnexpectedParameterInCaseSection("Unexpected `{0}` parameter. The `{1}` operator expects only one parameter."),

```

Comment:
I will try to find a more generic error code like

UnexpectedParameterWithSingleOperator("Unexpected `{0}` parameter. The `{1}` operator of the  `#{2}`  section  expects only one parameter.")

Reply:
You could maybe try to referring to it as an operand:

Unexpected operand `{0}`. The operator `{1}` in the section `{2}` is a unary prefix operator, and as such, accepts one operand.

The downside is this is a bit more technical, so it might be less obvious what the tooling is trying to communicate